src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -29,7 +29,9 @@ public enum DocumentFlags
         FromClusterTransaction = 0x1000,
         Reverted = 0x2000,
 
-        HasTimeSeries = 0x4000
+        HasTimeSeries = 0x4000,
+
+        ForceCreated = 0x10000
```

Comment:
let's be consistent with the `NonPersistentDocumentFlags` and name it `ForceRevisionCreation`

Reply:
Ping me about this tomorrow :)